[2025-06-20T08:30:52.763+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-20T08:30:52.779+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: twitter_etl_dag.load_to_azure manual__2025-06-20T08:28:35.001099+00:00 [queued]>
[2025-06-20T08:30:52.783+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: twitter_etl_dag.load_to_azure manual__2025-06-20T08:28:35.001099+00:00 [queued]>
[2025-06-20T08:30:52.783+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2025-06-20T08:30:52.789+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_to_azure> on 2025-06-20 08:28:35.001099+00:00
[2025-06-20T08:30:52.794+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=185) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-06-20T08:30:52.793+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'twitter_etl_dag', 'load_to_azure', 'manual__2025-06-20T08:28:35.001099+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/twitter_etl_dag.py', '--cfg-path', '/tmp/tmp73bsw9ej']
[2025-06-20T08:30:52.795+0000] {standard_task_runner.py:63} INFO - Started process 186 to run task
[2025-06-20T08:30:52.795+0000] {standard_task_runner.py:91} INFO - Job 19: Subtask load_to_azure
[2025-06-20T08:30:52.819+0000] {task_command.py:426} INFO - Running <TaskInstance: twitter_etl_dag.load_to_azure manual__2025-06-20T08:28:35.001099+00:00 [running]> on host 931f0b327f52
[2025-06-20T08:30:52.861+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='twitter_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_azure' AIRFLOW_CTX_EXECUTION_DATE='2025-06-20T08:28:35.001099+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-20T08:28:35.001099+00:00'
[2025-06-20T08:30:52.861+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-20T08:30:52.871+0000] {_universal.py:514} INFO - Request URL: 'https://twitterstorage123.blob.core.windows.net/tweets?restype=REDACTED'
Request method: 'PUT'
Request headers:
    'x-ms-version': 'REDACTED'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.19.1 Python/3.12.3 (Linux-6.10.14-linuxkit-aarch64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': 'e0bf6f0a-4db0-11f0-91f0-facd066c95b3'
    'Authorization': 'REDACTED'
No body was attached to the request
[2025-06-20T08:30:53.326+0000] {_universal.py:550} INFO - Response status: 409
Response headers:
    'Content-Length': '230'
    'Content-Type': 'application/xml'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': '7c8b30fd-701e-009d-40bd-e14068000000'
    'x-ms-client-request-id': 'e0bf6f0a-4db0-11f0-91f0-facd066c95b3'
    'x-ms-version': 'REDACTED'
    'x-ms-error-code': 'ContainerAlreadyExists'
    'Date': 'Fri, 20 Jun 2025 08:30:52 GMT'
[2025-06-20T08:30:53.329+0000] {twitter_etl_dag.py:68} INFO - Container 'tweets' exists or error: The specified container already exists.
RequestId:7c8b30fd-701e-009d-40bd-e14068000000
Time:2025-06-20T08:30:53.3378903Z
ErrorCode:ContainerAlreadyExists
Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.
RequestId:7c8b30fd-701e-009d-40bd-e14068000000
Time:2025-06-20T08:30:53.3378903Z</Message></Error>
[2025-06-20T08:30:53.331+0000] {_universal.py:511} INFO - Request URL: 'https://twitterstorage123.blob.core.windows.net/tweets/elonmusk_tweets.json'
Request method: 'PUT'
Request headers:
    'Content-Length': '2'
    'x-ms-blob-type': 'REDACTED'
    'x-ms-version': 'REDACTED'
    'Content-Type': 'application/octet-stream'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.19.1 Python/3.12.3 (Linux-6.10.14-linuxkit-aarch64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': 'e105a8b2-4db0-11f0-91f0-facd066c95b3'
    'Authorization': 'REDACTED'
A body is sent with the request
[2025-06-20T08:30:53.400+0000] {_universal.py:550} INFO - Response status: 201
Response headers:
    'Content-Length': '0'
    'Content-MD5': 'REDACTED'
    'Last-Modified': 'Fri, 20 Jun 2025 08:30:53 GMT'
    'ETag': '"0x8DDAFD4C546A712"'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': '7c8b3124-701e-009d-60bd-e14068000000'
    'x-ms-client-request-id': 'e105a8b2-4db0-11f0-91f0-facd066c95b3'
    'x-ms-version': 'REDACTED'
    'x-ms-content-crc64': 'REDACTED'
    'x-ms-request-server-encrypted': 'REDACTED'
    'Date': 'Fri, 20 Jun 2025 08:30:52 GMT'
[2025-06-20T08:30:53.401+0000] {twitter_etl_dag.py:73} INFO - âœ… Uploaded to Azure Blob Storage.
[2025-06-20T08:30:53.402+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-20T08:30:53.403+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-20T08:30:53.411+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=twitter_etl_dag, task_id=load_to_azure, run_id=manual__2025-06-20T08:28:35.001099+00:00, execution_date=20250620T082835, start_date=20250620T083052, end_date=20250620T083053
[2025-06-20T08:30:53.445+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-20T08:30:53.455+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-20T08:30:53.455+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
